packages/itwin/grouping-mapping-widget/src/widget/components/CustomCalculationAction.tsx

```diff
@@ -225,6 +226,13 @@ export const CustomCalculationAction = ({
               validator.showMessageFor("name");
             }}
           />
+          <Alert
+            type='informational'
+            clickableText='Click here.'
+            clickableTextProps={{ href: "https://developer.bentley.com/apis/insights/operations/create-customcalculation/", target: "_blank", rel: "noreferrer" }}
+          >
+            To learn more about creating custom calculation formulas, view the documentation.
```

Comment:
some slight changes in the tooltip text - capitalization and wording:
"To learn more about creating Custom Calculation formulas, click to view the documentation."

Reply:
Actually, can be more concise.

"Click to learn more about creating Custom Calculation formulas."

Reply:
Since the link text is "Click here.", is the beginning of this redundant. Then it would say:
Click to learn more about creating Custom Calculation formulas. Click here.

Reply:
I think maybe there was confusion because I no longer have it as a Tooltip. It is an Alert now.

Reply:
I see, could you update your recording to how it works now? I'm not able to picture how it looks now.

Reply:
Disregard last comment, I saw the recording in your other comment. How about:

"[Click here](www.google.com) to learn more about creating Custom Calculation formulas."

Reply:
What you have now is good too. I'll leave it up to you to decide.

Reply:
The way that the clickable text feature of the Alert works is that it automatically adds it to the end. I can just make it an anchor element instead, but I also like how it is now.